test/e2e/reachability.go

```diff
@@ -55,10 +55,28 @@ func (pod Pod) PodName() string {
 	return podName
 }
 
+type PodConnectivityMark uint8
```

Comment:
I don't see a reason for making this a `uint8` and not a `string`. This is testing code and if we need to log this value it will make things easier.

Reply:
Changed.